Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml

```diff
@@ -3557,25 +3572,22 @@ script:
       - resolved_true_positive
     - description: Comment to append to updated alerts.
       name: comment
-      required: false
     description: |-
       Update one or more alerts with the provided arguments.
       Required license: Cortex XDR Prevent, Cortex XDR Pro per Endpoint, or Cortex XDR Pro per GB.
     name: xdr-update-alert
   dockerimage: demisto/python3:3.11.9.107902
   isfetch: true
-  isfetch:xpanse: false
   script: ''
   subtype: python3
   ismappable: true
   isremotesyncin: true
   isremotesyncout: true
   type: python
+  runonce: false
 tests:
 - Test XDR Playbook execute script commands
 - Test XDR Playbook quarantine file command
 - Test XDR Playbook general commands
 - Test XDR Playbook retrieve file command
-defaultmapperin: Cortex XDR - IR-mapper
-defaultmapperout: Cortex XDR - IR-out-mapper
 fromversion: 5.0.0
```

Comment:
Please add unit test for the code you added under this file Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR_test.py.
[docs reference](https://xsoar.pan.dev/docs/integrations/unit-testing#write-your-unit-tests)

Reply:
@tcarmeli1 please clarify what unit tests are required. I didn't add any new functions or commands, I added a couple of arguments to a command and modified some existing functions/methods.

Reply:
Since new arguments were added, unit tests should be implemented to ensure that the previous functionality remains intact and that the new functionality works as expected. you can see examples in the current testing file
Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR_test.py.